litedbasync/LiteDatabaseAsync.cs

```diff
@@ -357,7 +329,10 @@ protected virtual void Dispose(bool disposing)
                     _shouldTerminate.Cancel();
                     
                     // give the thread 5 seconds to exit... must not block forever here
-                    _backgroundThread.Join(TimeSpan.FromSeconds(5));
+                    if (!_backgroundThread.Join(TimeSpan.FromSeconds(5)))
+                    {
+                        var tooslow = true;
```

Comment:
the tooslow variable is unused. If you are going to use it it should be in camelCase eg tooSlow

Reply:
Removed the variable